src/main/java/org/mariadb/jdbc/util/DefaultOptions.java

```diff
@@ -493,6 +494,15 @@
           + "deactivating this validation. Hostname verification is disabled when the trustServerCertificate "
           + "option is set",
       false),
+  ENABLE_REDIRECTION(
+      "enableRedirect",
+      RedirectionOption.OFF.toString().toLowerCase(),
+      "2.1.0",
+      "This option let user to decide if use redirection feature and chose different options: "
+          + "\n off: Do not use redirection."
+          + "\n on: Must use redirection, fail the connection if server doesnot support redirection or redirection fail."
+          + "\n preferred: Use redirection if redirection and redirection is success, otherwise do not use redirection.",
```

Comment:
```suggestion
          + "\n preferred: Use redirection if supported by the server and redirecting is successful; otherwise, do not use redirection.",
```